cohorts/load.py

```diff
@@ -395,7 +481,7 @@ def load_variants(self, variant_type="snv", merge_type="union"):
         assert variant_type in ["snv", "indel"], "Unknown variant type: %s" % variant_type
         patient_variants = {}
 
-        for patient in self:
+        for patient in self.iter_patients(patients):
```

Comment:
It handles the default of `patients=None` so that will use `self`; couldn't think of a cleaner way, since I can't put `self` as the default for `patients` in the method signatures.